admin/build-doc

```diff
@@ -22,7 +22,11 @@ if command -v dpkg >/dev/null; then
 elif command -v yum >/dev/null; then
     for package in python-devel python-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
 	if ! rpm -q $package >/dev/null ; then
-		missing="${missing:+$missing }$package"
+		command -v `echo $package | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'`   > /dev/null;
```

Comment:
@alfredodeza
This is not the exact command that will be executed
``` 
$ rpm -q ant | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'
1.9.2
```

The actual command that will be executed is 
```
$ command -v `echo $package | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'`   > /dev/null;
$ <Returns Nothing>
```


However, for 'ant' package it will not reach above new execution, as the following if condition will not be true and thus 'command' won't execute.
```
24     if ! rpm -q ant >/dev/null ; then

$ rpm -q ant
ant-1.9.6-3.fc24.noarch
```

> The problem here is that the package names are not matching the ones installed in the system. It isn't `python-virtualenv` in your FC system, it is python2-virtualenv.

Yes, agreed that is what I intended to resolve. So that  it should not fail even if the package is installed.

```
$ dnf install -y python-virtualenv
Last metadata expiration check: 1:06:54 ago on Tue Aug 22 01:11:37 2017.
Package `python2-virtualenv-15.0.3-2.fc25.noarch` is already installed, skipping.
Dependencies resolved.
Nothing to do.
Complete!
```

Let me know your views.